frontend/src/metabase/visualizations/lib/save-chart-image.ts

```diff
@@ -28,6 +28,7 @@ export const saveChartImage = async (selector: string, fileName: string) => {
 
   const { default: html2canvas } = await import("html2canvas-pro");
   const canvas = await html2canvas(node, {
+    scale: 2,
```

Comment:
based on https://github.com/niklasvh/html2canvas/issues/2302#issuecomment-672846820, this is how we ake the export consistent and without weird background colors.

Reply:
it doesn't seem to affect other screens, Macbook/iPad seems to have the effect of `scale: 2` already so this setting does nothing to them.